llama2.mojo

```diff
@@ -871,6 +884,7 @@ fn print_usage():
     print("  -n <int>    number of steps to run for, default 256. 0 = max_seq_len")
     print("  -i <string> input prompt")
     print("  -z          tokenizer path")
+    print("  -a <string> architecture, default 'llama2c' for tinystories, 'tinyllama' for hf tinyllama-1.1")
```

Comment:
Could you change this config to 0 and 1?
Since tinystories and tinyllama are model but not the rope type.

0->llama2c  rotate pair of even and odd dimension
1->tinyllama roate first half and second half

Reply:
Don't those methods have any human readable names?

Reply:
No they don't. Maybe we can use more famous model name to represent the rope type.

llama(rotate pair of even and odd dimension) -> llama2c
falcon(roate first half and second half) -> tinyllama

Reply:
I have the -a to -r which means the rope architecture.
And default is `llama` representing the tinystories rope. The other value is `falcon`.